src/analysis/retail/deathknight/frost/Guide.tsx

```diff
@@ -0,0 +1,125 @@
+      <Section title="Cooldowns">
+        <CooldownsSubsection modules={modules} events={events} info={info} />
+      </Section>
+    </>
+  );
+}
+
+function CooldownsSubsection({ modules, events, info }: GuideProps<typeof CombatLogParser>) {
+  return (
+    <SubSection>
+      <strong>Cooldown Graph</strong> - this graph shows when you used your cooldowns and how long
+      you waited to use them again. Grey segments show when the spell was available, yellow segments
+      show when the spell was cooling down. Red segments highlight times when you could have fit a
+      whole extra use of the cooldown.
+      {info.combatant.hasTalent(talents.PILLAR_OF_FROST_TALENT) && (
+        <div className="flex-main chart" style={{ padding: 5 }}>
+          <CooldownBar
+            spellId={talents.PILLAR_OF_FROST_TALENT.id}
+            events={events}
+            info={info}
+            highlightGaps
+          />
+        </div>
+      )}
+      {info.combatant.hasTalent(talents.REMORSELESS_WINTER_TALENT) && (
+        <div className="flex-main chart" style={{ padding: 5 }}>
+          <CooldownBar
+            spellId={talents.REMORSELESS_WINTER_TALENT.id}
+            events={events}
+            info={info}
+            highlightGaps
+          />
+        </div>
+      )}
+      {info.combatant.hasTalent(talents.CHILL_STREAK_TALENT) && (
+        <div className="flex-main chart" style={{ padding: 5 }}>
+          <CooldownBar spellId={talents.CHILL_STREAK_TALENT.id} events={events} info={info} />
+        </div>
+      )}
+      {info.combatant.hasTalent(talents.BREATH_OF_SINDRAGOSA_TALENT) && (
+        <div className="flex-main chart" style={{ padding: 5 }}>
+          <CooldownBar
+            spellId={talents.BREATH_OF_SINDRAGOSA_TALENT.id}
+            events={events}
+            info={info}
+            highlightGaps
+          />
+        </div>
+      )}
+      {info.combatant.hasTalent(talents.FROSTWYRMS_FURY_TALENT) && (
+        <div className="flex-main chart" style={{ padding: 5 }}>
+          <CooldownBar
+            spellId={talents.FROSTWYRMS_FURY_TALENT.id}
+            events={events}
+            info={info}
+            highlightGaps
+          />
+        </div>
+      )}
+      {info.combatant.hasTalent(talents.HORN_OF_WINTER_TALENT) && (
+        <div className="flex-main chart" style={{ padding: 5 }}>
+          <CooldownBar
+            spellId={talents.HORN_OF_WINTER_TALENT.id}
+            events={events}
+            info={info}
+            highlightGaps
+          />
+        </div>
+      )}
+      {(info.combatant.hasTalent(talents.EMPOWER_RUNE_WEAPON_FROST_TALENT) ||
+        info.combatant.hasTalent(talents.EMPOWER_RUNE_WEAPON_SHARED_TALENT)) && (
+        <div className="flex-main chart" style={{ padding: 5 }}>
+          <CooldownBar
+            spellId={spells.EMPOWER_RUNE_WEAPON.id}
+            events={events}
+            info={info}
+            highlightGaps
+          />
+        </div>
```

Comment:
I recently merged a refactor of `CooldownBar` that makes it require less stuff - you'll have to carry through the refactor to these usages for them to work once its merged. I recommend pulling from dragonflight.

The new usage will be: ```
<CooldownBar
  spellId={ ... }
  gapHighlightMode={GapHighlight.FullCooldown}
/>```